tox.ini

```diff
@@ -29,3 +29,17 @@ commands = pep8 --ignore E501 luigi test examples bin
 [testenv:autopep8]
 deps = autopep8
 commands = autopep8 --ignore E501 -a -i -r luigi test examples bin
+
+[testenv:docs]
+# to call this use `tox -e docs -- html`
+deps =
+  sqlalchemy
+  Sphinx
+  sphinx_rtd_theme
+
+changedir = doc
+
+commands =
+    make {posargs}
```

Comment:
@Tarrasch it seems `{posargs: ...}` is used when you want to set defaults to positional arguments.

http://tox.readthedocs.org/en/1.0/example/general.html